uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
> Come to think of it, I was assuming we would want volunteer signup in at-the-door reg, but come to think of it we just tell people to come to Stops, so I should probably remove that as well. I'll handle that in a separate commit in this branch in a few minutes.

This has been done, it's the same line below (which I already commented on) as the one that removes the kick-in rows.